test/integration/test_session.py

```diff
@@ -91,9 +91,9 @@ def test_session_from_id(self, service):
             mock_create_session.assert_not_called()
 
         self.assertEqual(session._session_id, new_session._session_id)
-        self.assertTrue(new_session._active)
```

Comment:
This assertion should hold in the new IQP too, right? the session should be active before calling `.close()`? Or has the behavior of `._active` changed?

Reply:
Yeah this holds for the new IQP but fails on IQP classic because once the job completes the session closes right away. 

Not sure if it's some sort of timing discrepancy but this is the current behavior ([from_id code](https://github.com/kt474/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/session.py#L353)): 

Old IQP - job in session completes -> `from_id()` new session -> new session has "state" of `closed` so `._active` is false

New IQP - job in session completes -> `from_id()` new session -> new session has "state" of `active` so `._active` is true